tests/rbd_mirror/test-add-remove-osd-with-mirroring.py

```diff
@@ -0,0 +1,224 @@
+from ceph.parallel import parallel
+from tests.rbd_mirror.rbd_mirror_utils import execute_dynamic, rbd_mirror_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_add_remove_osd_with_mirroring(rbd_mirror, pool_type, **kw):
+    """
+    Test to add and remove osd nodes with mirroring
+    Args:
+        rbd_mirror: rbd mirror object
+        pool_type: ec pool or rep pool
+        **kw:
+
+    Returns:
+        0 if test passes, else 1
+    """
+    try:
+        mirror1 = rbd_mirror.get("mirror1")
+        mirror2 = rbd_mirror.get("mirror2")
+        config = kw.get("config")
+        pool = config[pool_type]["pool"]
+        image = config[pool_type]["image"]
+        imagespec = pool + "/" + image
+
+        results = {}
+
+        log.info("Add OSDs to primary cluster with bench IOs running on mirrored image")
+        with parallel() as p:
+            p.spawn(
+                execute_dynamic,
+                mirror1,
+                "benchwrite",
+                results,
+                imagespec=imagespec,
+                io="100M",
```

Comment:
Can we please increase the IO sizes to 1G ?

Reply:
I added small IOs since I want to run IO only till the add or remove osd operation is in progress. If we have large IOs then entries behind primary is taking forever to get back to 0.
Please let me know how to proceed

Reply:
Okay with the intention.